picamera2/request.py

```diff
@@ -168,8 +166,10 @@ def save_dng(self, filename, name="raw"):
 
 
 class Helpers:
-    """This class implements functionality required by the CompletedRequest methods, but
-    in such a way that it can be usefully accessed even without a CompletedRequest object."""
+    """This class implements functionality required by the CompletedRequest methods.
+
+    In such a way that it can be usefully accessed even without a CompletedRequest object.
```

Comment:
Did flake8 reword this and change the grammar? It's obviously magic (though I'm not totally sure about the result)!

Reply:
Sorry I had had to do that manually, so there's a summary line. I agree that needs improving how I've done that.